src/briefcase/config.py

```diff
@@ -191,6 +191,83 @@ def validate_document_type_config(document_type_id, document_type):
         pass
 
 
+def validate_install_options_config(config):
+    """Validate that a install options are valid and complete, and convert to a dict.
+
+    The dict format is required because Cookiecutter doesn't allow passing a list as a
+    context value; you have to use the reliable iteration order of a dict instead.
+    """
+    install_options = {}
+    known_names = set()
+    if config:
+        for i, config_item in enumerate(config):
+            try:
+                name = config_item["name"]
+                if not isinstance(name, str):
+                    raise BriefcaseConfigError(
+                        f"Name for install option {i} is not a string."
+                    ) from None
```

Comment:
`from None` is unnecessary here because it's not inside an `except` block. There are several other instances in this function.